src/tools/ceph_monstore_tool.cc

```diff
@@ -927,10 +945,13 @@ int main(int argc, char **argv) {
         ::remove(outpath.c_str());
       }
     });
-
     bufferlist bl;
     r = 0;
-    if (map_type == "osdmap") {
+    if (map_type == "osd_snap") {
+      std::cout << map_type << " " << key << std::endl;
+      // map_type is the prefix
```

Comment:
Instead of overloading `get <map_type>` command which is intended for maps and a) remembering that `map_type` is really not a map type and b) potentially returning error messages such as "Error getting map ..." or "This type of readable map does not exist ..." from a command that isn't related to maps at all, I would introduce a new `get-key` command.  The `rm` command that is also being added here could be renamed to `rm-key` or `remove-key` to match.